### ISSUE TITLE
Stripe initialization

### ISSUE THREAD

**Issue by t-vladimir** [capacitor-community/stripe#222]
Hello.

**Platform**
- [ ] Web
- [x] iOS
- [ ] Android

**Describe the bug**
When I'm redirected by a router to the page with payments, I'm trying to initialize Stripe, and the IOS emulator outputs:
`⚡️  To Native ->  Stripe initialize 37184741`
`⚡️  TO JS undefined`

**To Reproduce**
`Stripe.initialize({publishableKey: VUE_APP_STRIPE_PUBLIC_KEY})`

**Screenshots**
<img width="319" alt="image" src="https://user-images.githubusercontent.com/111683680/199066254-773850b1-847c-441b-b8cd-6fb58c94b598.png">

**Additional context**
I'm using Ionic and Vuejs.

`@capacitor-community/stripe": "^4.1.0`

I was trying to find the problem with existing issues here and found [this ticket](https://github.com/capacitor-community/stripe/issues/85) and there was the same problem, but the issuer said that he had another problem, it was about the ephemeral key. (I'm certain issuer has resolved this issue in another way but that ticket has no information about it)

Still, I did the same rdlabo requested to do - I've added the `defineCustomElements(window);` in a way that Ionic Vuejs
[demands](https://stackoverflow.com/questions/64695854/where-to-definecustomelementswindow-in-ionic-vue-photogallery-app) and still no results.

I was also trying to go further and have called `isApplePayAvailable()` but it just hangs and never return any values at `then()/catch()`

**Comment by t-vladimir** [comment]
Hello?
Could anyone guide what I'm doing wrong?

**Comment by t-vladimir** [comment]
So, accidentally I found the problem, the reason was IOS Emulator, which demanded to confirm Apple ID.
When I was starting the app, the iPhone emulator was asking to confirm my Apple account with a popup, for some strange reason when I was clicking on the button to confirm it, iPhone was just opening Settings, and that's all, no screen to input the password.
But for another strange reason (I really don't know how that happen) I got another popup after I was redirected to Settings, on which I clicked, and then it showed me the input for the password, I typed it and it worked without errors.
